test/lib/ansible_test/_data/sanity/import/importer.py

```diff
@@ -481,13 +482,24 @@ def capture_output(capture):
                     "ignore",
                     "Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography,"
                     " and will be removed in the next release.")
+
             if sys.version_info[:2] == (3, 5):
                 warnings.filterwarnings(
                     "ignore",
                     "Python 3.5 support will be dropped in the next release ofcryptography. Please upgrade your Python.")
                 warnings.filterwarnings(
                     "ignore",
                     "Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.")
+
+            if sys.version_info >= (3, 10):
+                # Temporary solution for Python 3.10 until find_spec is implemented in RestrictedModuleLoader.
+                # That implementation is dependent on find_spec being added to the controller's collection loader first.
+                # The warning text is: main.<locals>.RestrictedModuleLoader.find_spec() not found; falling back to find_module()
+                warnings.filterwarnings(
+                    "ignore",
+                    r"main\.<locals>\.RestrictedModuleLoader\.find_spec\(\) not found; falling back to find_module\(\)",
```

Comment:
nit: I'd also narrow this down with a module name where this is allowed to happen + exception type